google-oauth-client-java6/src/main/java/com/google/api/client/extensions/java6/auth/oauth2/AuthorizationCodeInstalledApp.java

```diff
@@ -110,7 +152,15 @@ protected void onAuthorization(AuthorizationCodeRequestUrl authorizationUrl) thr
    * @throws IOException I/O exception
    */
   protected void onAuthorization(AuthorizationCodeRequestUrl authorizationUrl) throws IOException {
-    browse(authorizationUrl.build());
+    String url = authorizationUrl.build();
+    Preconditions.checkNotNull(url);
+    System.out.println("Please open the following address in your browser:");
```

Comment:
We shouldn't need these two `println` statements.

Reply:
Will remove those.